yak/package.xml

```diff
@@ -12,6 +12,7 @@
   <depend>eigen</depend>
   <depend>nvidia-cuda-dev</depend>
   <depend>libopencv-dev</depend>
+  <depend>libomp-dev</depend>
```

Comment:
:+1: for submitting the PR, but is `libomp-dev` a `rosdep` key? [base.yaml](https://github.com/ros/rosdistro/blob/866f3deed3bdcad60c0399e683fdd813ecbf2e26/rosdep/base.yaml) does not seem to contain it.

Reply:
Ah, I see! I'll have a look at the `rosdep` [contributing rules](http://docs.ros.org/independent/api/rosdep/html/contributing_rules.html) tomorrow to see if it can be added and how much effort that takes :).

Reply:
:bellhop_bell: ;)

Reply:
@dave992 Any further developments on this?

Reply:
I'll pick this up again, thanks for the reminder

Reply:
I've submitted a PR to add the rosdep key: https://github.com/ros/rosdistro/pull/25553

Reply:
libomp-dev is now a rosdep key :)